addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -105,13 +105,7 @@ var SlideUploadDialog = Dialog.extend({
         var self = this;
         this.$('#category_id').select2(this._select2Wrapper(_t('Section'), false,
             function () {
-                return self._rpc({
-                    route: '/slides/category/search_read',
-                    params: {
-                        fields: ['name'],
-                        domain: [['channel_id', '=', self.channelID]],
-                    }
-                });
+                return self._fetchSections()
             })
```

Comment:
no need to wrap _fetchSections into an other function
```suggestion
        self._fetchSections